<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hey! Healer - 헤이딜러</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="q_heydealer_login_group" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>440</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <widget class="QLabel" name="q_login_text">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>헤이딜러에 접속된 아이디</string>
     </property>
    </widget>
    <widget class="QLabel" name="q_user_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="q_process_group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>440</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>프로세스</string>
    </property>
    <widget class="QGroupBox" name="q_favorite_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="title">
      <string>1. 관심차량 프로세스</string>
     </property>
     <widget class="QCheckBox" name="q_favorite_enable_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="q_favorite_progressbar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>23</y>
        <width>200</width>
        <height>24</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="q_favorite_start_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>시작</string>
      </property>
     </widget>
     <widget class="QPushButton" name="q_favorite_stop_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>중지</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="q_favorite_ended_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="title">
      <string>2. 찜 종료 프로세스</string>
     </property>
     <widget class="QCheckBox" name="q_favorite_ended_enable_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="q_favorite_ended_progressbar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>23</y>
        <width>200</width>
        <height>24</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="q_favorite_ended_start_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>시작</string>
      </property>
     </widget>
     <widget class="QPushButton" name="q_favorite_ended_stop_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>중지</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="q_auction_ended_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="title">
      <string>3. 내입찰 완료 프로세스</string>
     </property>
     <widget class="QCheckBox" name="q_auction_ended_enable_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="q_auction_ended_progressbar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>23</y>
        <width>200</width>
        <height>24</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="q_auction_ended_start_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>시작</string>
      </property>
     </widget>
     <widget class="QPushButton" name="q_auction_ended_stop_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>중지</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="q_auction_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>420</width>
       <height>60</height>
      </rect>
     </property>
     <property name="title">
      <string>4. 입찰 프로세스</string>
     </property>
     <widget class="QCheckBox" name="q_auction_enable_check">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>24</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="q_auction_progressbar">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>23</y>
        <width>200</width>
        <height>24</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="q_auction_start_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>시작</string>
      </property>
     </widget>
     <widget class="QPushButton" name="q_auction_stop_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>중지</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="q_process_start_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="q_process_stop_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>중지</string>
     </property>
    </widget>
    <widget class="QLabel" name="q_process_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>33</y>
       <width>200</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>선택된 프로세스를</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="q_url_group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>440</width>
      <height>100</height>
     </rect>
    </property>
    <property name="title">
     <string>URL 분석</string>
    </property>
    <widget class="QPushButton" name="q_url_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>차량 분석 및 데이터 수집</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="q_url">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>400</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>설정</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QCheckBox" name="q_limit_scroll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>최대 스크롤 갯수 제한</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="q_limit_scroll_count">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>60</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>230</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QSpinBox" name="q_crawling_start_index">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>60</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>크롤링 시작 인덱스</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>스크롤 대기 시간</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="q_wait_scroll_time">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>60.000000000000000</double>
      </property>
      <property name="value">
       <double>15.000000000000000</double>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>q_process_start_button</tabstop>
  <tabstop>q_process_stop_button</tabstop>
  <tabstop>q_favorite_enable_check</tabstop>
  <tabstop>q_favorite_start_button</tabstop>
  <tabstop>q_favorite_stop_button</tabstop>
  <tabstop>q_favorite_ended_enable_check</tabstop>
  <tabstop>q_favorite_ended_start_button</tabstop>
  <tabstop>q_favorite_ended_stop_button</tabstop>
  <tabstop>q_auction_ended_enable_check</tabstop>
  <tabstop>q_auction_ended_start_button</tabstop>
  <tabstop>q_auction_ended_stop_button</tabstop>
  <tabstop>q_auction_enable_check</tabstop>
  <tabstop>q_auction_start_button</tabstop>
  <tabstop>q_auction_stop_button</tabstop>
  <tabstop>q_url</tabstop>
  <tabstop>q_url_button</tabstop>
  <tabstop>q_limit_scroll</tabstop>
  <tabstop>q_limit_scroll_count</tabstop>
  <tabstop>q_crawling_start_index</tabstop>
  <tabstop>q_wait_scroll_time</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
